include/cuda/std/detail/libcxx/include/optional

```diff
@@ -700,76 +761,82 @@ public:
     constexpr explicit optional(_InPlaceT, _Args&&... __args)
         : __base(in_place, _CUDA_VSTD::forward<_Args>(__args)...) {}
 
-    template <class _Up, class... _Args, class = _EnableIf<
+    template <class _Up, class... _Args, class = __enable_if_t<
```

Comment:
I had all kinds of funny issues sometimes with plain `__enable_if_t`

Should we move to Erics concept emulation?

That would also be slightly more readable

Reply:
I haven't seen anything in the CI, so I'll leave it as is and fix if we do somehow get issues.